modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectValidationRuleLocalServiceImpl.java

```diff
@@ -226,6 +226,16 @@ public List<ObjectValidationRule> getObjectValidationRules(
 				objectDefinitionId, active));
 	}
 
+	@Override
+	public List<ObjectValidationRule>
+		getObjectValidationRulesByObjectValidationRuleEngine(
```

Comment:
@igor-franca please rename it to `getObjectValidationRules` if possible. If not, rename it to `getObjectValidationRulesByODI_E`.